Gemfile

```diff
@@ -250,8 +250,6 @@ group :development, :test do
   gem 'pry-shell'
   gem 'pry-stack_explorer'
 
-  # for `rake doc:liquid:generate` and similar
```

Comment:
Is this relevant? Should we remove/edit this task?

Reply:
I think this was a "typo" and it referred to `doc:swagger:generate` which I already removed.

Reply:
Do we still need `swagger-ui_rails`?

Reply:
> Do we still need `swagger-ui_rails`?

I'm afraid we do. I believe this is for Swagger v1, and we still support rendering Swagger UI for v1 specs for our customers.